index.html

```diff
@@ -17,12 +18,196 @@
 <body>
 
     <div class="container">
-      <div class="row">
+      
+      <div class="row my-4">
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
+            <ul class="nav justify-content-end ">
+              <li class="nav-item">
+                <a class="nav-link active" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#skills">Skills</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#interests">Interests</a>
+              </li>
+              <li class="nav-item dropdown">
+                  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Helpful Links</a>
+                  <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                      <a class="dropdown-item" href="http://lesscss.org/" target="_blank">LESS Documentation</a>
+                      <a class="dropdown-item" href="https://getbootstrap.com/docs/4.0/getting-started/introduction/" target="_blank">Bootstrap 4 Documentation</a>
+                      <a class="dropdown-item" href="https://developer.mozilla.org/en-US/" target="_blank">Mozilla Developer Network</a>
+                    </div>
+              </li>
+            </ul>
+          </div>
+      </div>
+    
+
+    
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="jumbotron">
+            <img class="jumboPic img-fluid rounded-circle" src="img/kepf.jpg" alt="">
+            <h1 class="display-4">Welcome to Jeremy's Online Resume</h1>
+            <p class="lead">Every day is a new challenge, and I wouldn't have it any other way.</p>
+            <hr class="my-4">
+            
+            <p class="lead">
+              <a class="btn btn-primary btn-lg" href="#" role="button">Contact Me</a>
+            </p>
```

Comment:
It could be neat if you added a contact form at the bottom of the page that this button navigated the user to, or possibly a pop-up form.